cypress/integration/me.ts

```diff
@@ -0,0 +1,18 @@
+describe('me', () => {
+  it('shows the timeline', () => {
+    cy.login()
+    cy.visit('/me')
+
+    // shows timeline items
+    cy.get('body').should('contain', 'You created cypress+1’s Team')
```

Comment:
I was looking at the video of this test:

![vid](https://builds.cypress.io/d85ba524-7fc7-47cf-82ea-a608bde1e996?AWSAccessKeyId=AKIAIGH7VO3KJJU4LBGQ&Expires=1551892620&Signature=FxHwwupTTrWthVvpZiddr96o52w%3D&response-content-disposition=attachment%3B%20filename%3D%22cypress-video.mp4%22%3B&response-content-type=video%2Fmp4)

And I was like 🤔 why don't I see the timeline? Looking at the test source, I see it's also navigating to the created team. I feel like if the test is supposed to test navigating to the the timeline, it should sorta stop there. But, you know, it's probably fine.

Reply:
agree! each file should contains logic for a single view/route. IMO, to test that a link works on a particular route, that means i need to test clicking it (https://www.youtube.com/watch?v=5XQOK0v_YRE). Yep, we'll see the team page pop up, but that's it. we don't test any logic on that page, we just test that clicking something on /me works

Reply:
Totally makes sense. I just expected to see the test finish on the timeline (i.e. I could see a timeline with my human eyeballs) but Cyprus moves too fast :)

Reply:
if you run it locally you totally can! i think they skip frames when they upload the video